### PR TITLE
Fixes for slider

### PR DESCRIPTION
**Pull request by a2batic**
1) Added animation to the first three logos.
2) Added signifies on hover to showcase hyperlinks.
3) Removed rounded corner.
4) Tested for responsiveness

### CODE REVIEW

File: src/client/css/style.css [hunk 1]
```diff
@@ -458,15 +460,20 @@ section#community .fa-globe{
     display: block;
     margin: 0;
     position: relative;
+    border-radius: 0;
+
     -webkit-transition: background-position .5s ease-in-out;
     -moz-transition: background-position .5s ease-in-out;
     -o-transition: background-position .5s ease-in-out;
-    -ms-transition: background-position .5s ease-in-out;
+    -ms-transition: background-position 5s ease-in-out;
```
Comment (handasolo) [inline src/client/css/style.css]: Is this intentional?

**Comment by handasolo** [comment]
Bit big too review. Might take some time.

File: src/client/css/style.css (hunk 1 above)
Comment (a2batic) [inline src/client/css/style.css]: Mistake-No... It should be transition only.. !!

Comment (a2batic) [inline src/client/css/style.css]: Mistake-And it was .5s instead of 5s !!

**Comment by handasolo** [comment]
You should try to make the animations of the first three slider more smooth. Otherwise, the code looks good to go.

**Pull request closed without merging**